test/ttmlir/Dialect/TTNN/linear/linear_tests_positive.mlir

```diff
@@ -1,18 +1,5 @@
 // RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline %s | FileCheck %s
 module {
-  func.func @linear_1d_1d(%arg0: tensor<128xbf16>, %arg1: tensor<128xbf16>) -> tensor<1xbf16> {
```

Comment:
There have been a few tests that have failed due to canonicalization changing the graph. I have removed them and wrote tests that specifically check for the correctness of rewriting.